bindings/matrix-sdk-crypto-ffi/src/backup_recovery_key.rs

```diff
@@ -161,10 +164,16 @@ impl BackupRecoveryKey {
     pub fn decrypt_v1(
         &self,
         ephemeral_key: String,
-        mac: String,
+        macv1: Option<String>,
+        macv2: Option<String>,
```

Comment:
Not really a fan of this function signature, wasn't a fan before this PR either.

I added a different one here: https://github.com/matrix-org/matrix-rust-sdk/blob/2f41fe1e96ea567d9ab66366cacc907523db7585/crates/matrix-sdk-crypto/src/backups/keys/decryption.rs#L225-L242.